arangod/Pregel/Algos/SLPA.h

```diff
@@ -40,6 +40,56 @@ namespace algos {
 /// interaction  rules.  It has excellent performance in identifying both
 /// overlapping
 /// nodes and  overlapping  communities  with  different  degrees  of diversity.
+
+/**
+ * From SLPA: Jierui Xie, Boleslaw K. Szymanski, Xiaoming Liu. Uncovering
+ * Overlapping Communities in Social Networks via A Speaker-listener Interaction
+ * Dynamic Process, 2011.
+ *
+ * The algorithm should actually be the following. In the paper, it is kept
+ * much more general. Our intention of the implementation of the places
+ * described generally in the paper is based on the ArangoDB's documentation.
+ *
+ * The algorithm is performed in a series of iterations during that vertices
+ * send vertex IDs (natural numbers) to each other. Each vertex contains all
+ * vertex IDs it has received since the beginning together with the information
+ * how often an ID has been received. At the beginning, a vertex contains only
+ * its own ID that was "obtained" once. In an iteration, the vertices are
+ * processed in a random order, which is chosen for each iteration separately.
+ * A vertex v gets from each of its in-neighbors one of their saved IDs. Which
+ * if the saved IDs is sent by an in-neighbor n is chosen randomly with the
+ * probability
+ *
+ * <number of times the ID was obtained by n> / <number of times any ID
+ * was obtained by n>.
+ *
+ * Vertex n chooses one of the received IDs to save in the
```

Comment:
I'm confused with this n. Before, n was the sending vertex. Perhaps better to use v here?

Reply:
It is a _n_eighbor of v. Would w be better?

Reply:
I applied it, please, check along with the other changes.